lib/actions/api.js

```diff
@@ -644,9 +700,21 @@ export function findRoutes(params) {
     findRoutesResponse,
     findRoutesError,
     {
-      rewritePayload: (payload) => {
+      postprocess: (payload, dispatch) => {
+        return dispatch(findRoutesV2())
```

Comment:
This call is failing both on existing OTPv2 servers and local OTPv2 server that use the dev 2.x branches, and thus the FLEX routes that are on the OTPv2 server but not on the other one are not shown in the route list. Are there commits required to make that work?

Reply:
(The comment does not apply if only a v2 server is used.)

Reply:
I'm unable to replicate this. As you can see in this screenshot, 2 separate OTP instances are able to be combined. This is using the latest commit of this branch and the latest commit of the `ibi-dev-2.x` OTP branch. Did you enable `"SandboxAPITransmodelApi": true,
  	"SandboxAPILegacyGraphQLApi": true,
` in `otp-config.json`? I think it's also possible that the `pbf` file needs to be there even to just list out the routes.

<img width="465" alt="Screen Shot 2021-12-30 at 10 56 13 AM" src="https://user-images.githubusercontent.com/86619099/147741182-d09ab8d3-29e7-4ab4-a8d5-6486ff56b8b0.png">